bitcoin/src/blockdata/script/borrowed.rs

```diff
@@ -350,7 +369,7 @@ impl Script {
     /// (Note: taproot scripts don't count toward the sigop count of the block,
     /// nor do they have CHECKMULTISIG operations. This function does not count OP_CHECKSIGADD,
     /// so do not use this to try and estimate if a taproot script goes over the sigop budget.)
-    pub fn count_sigops(&self) -> usize { self.count_sigops_internal(true) }
+    pub fn count_sigops_accurate(&self) -> usize { self.count_sigops_internal(true) }
```

Comment:
In b5347f8d754d6c2daa287d838af5dcfde1910cad:

I think the name `count_sigops` is fine. The doccomment which discusses "accerate" vs "legacy" can be left unchanged.